docs/source/prototype_models.rst

```diff
@@ -0,0 +1,23 @@
+.. _prototype_models:
+
+Prototype Models
+################
```

Comment:
When the porting from prototype to main is done, this will just become "Models" and the `prototype_models.rst` file will just become (and overwrite) `models.rst`